src/app/lib/utilities/episodeAvailability/index.test.js

```diff
@@ -8,98 +9,244 @@ import onDemandRadioBrandJson from '#data/indonesia/bbc_indonesian_radio/w13xtt0
 import onDemandTvEpisodeJson from '#data/afrique/bbc_afrique_tv/w13xttmz.json';
 import onDemandTvBrandJson from '#data/somali/bbc_somali_tv/tv_programmes/w13xttqt.json';
 
-describe('Logging episodeAvailability', () => {
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
+describe('Episode Availability', () => {
+  describe('getUrl', () => {
+    it('should return a valid url for radio episode page', () => {
+      const url = getUrl(onDemandRadioEpisodeJson);
+      const service = path(
+        ['relatedContent', 'site', 'name'],
+        onDemandRadioEpisodeJson,
+      ).toLowerCase();
```

Comment:
the `toLowerCase()` is negating the use of `path` here - path will safely return undefined, but calling a function on undefined will cause an exception anyway 🙂

Reply:
Who wrote this????!!!! I will remove

Reply:
When this is fixed I think it's good to 🚢 ?